tileserver/__init__.py

```diff
@@ -38,12 +40,26 @@ def coord_is_valid(coord):
     return True
 
 
-RequestData = namedtuple('RequestData', 'layer_spec coord format')
+RequestData = namedtuple('RequestData', 'layer_spec coord format params')
```

Comment:
I tried, and failed, to think of a better word for this. To be honest, I don't like `style` or `params`...

This is a variable which can inject variables into the `RequestData` object based on URL path prefixes. At the moment, only `tile_size` is used... so am I being too abstract? Should I just make it `tile_size`?